scripts/upgrade

```diff
@@ -12,65 +12,84 @@ source /usr/share/yunohost/helpers
 #=================================================
 # LOAD SETTINGS
 #=================================================
+ynh_script_progression --message="Loading installation settings..." --weight=2
 
 app=$YNH_APP_INSTANCE_NAME
 
-domain=$(ynh_app_setting_get $app domain)
-path_url=$(ynh_app_setting_get $app path)
-port=$(ynh_app_setting_get $app port)
-peer_port=$(ynh_app_setting_get $app peer_port)
+domain=$(ynh_app_setting_get --app=$app --key=domain)
+path_url=$(ynh_app_setting_get --app=$app --key=path)
+port=$(ynh_app_setting_get --app=$app --key=port)
+peer_port=$(ynh_app_setting_get --app=$app --key=peer_port)
 
 #=================================================
-# FIX OLD THINGS
+# CHECK VERSION
 #=================================================
 
-if [ -z $port ]; then
+upgrade_type=$(ynh_check_app_version_changed)
+
+#=================================================
+# ENSURE DOWNWARD COMPATIBILITY
+#=================================================
+ynh_script_progression --message="Ensuring downward compatibility..." --weight=4
+
+if [ -z "$port" ]; then
 	port=9091
```

Comment:
I know you just normalize scripts, but this line can be a problem if this port is already used.

To replace with `port=$(ynh_find_port --port=9091)`

Do you want I create a new PR for that or it's a small change and we can commit in this one?

Reply:
I noticed that myself while working on this script. I thought, since it's a fix for old scripts, that it was probably because when the install script wasn't setting the port into the config, it was probably as well using a fixed port.
I didn't check though...